bindings/protocols/coap/index.template.html

```diff
@@ -210,6 +210,56 @@ <h3>Requests/Responses</h3>
     }
 }
 </pre>
+            <p>
+                In CoAP over UDP or DTLS [[RFC7252]], requests and responses can either be confirmable (CON) or non-confirmable (NON).
+                In contrast to non-confirmable messages, confirmable messages are required to be acknowledged by the receiving endpoint and are otherwise retransmitted until an acknowledgement has been received.
+                Using non-confirmable requests can be useful for sensor data (that gets sent repeatedly and therefore does not need to be transmitted reliably), while confirmable requests provide a certain reliability that is useful for retrieving values or invoking actions that should not fail.
+            </p>
+            <p>
+                To provide a hint whether CON or NON requests should be used for accessing a CoAP resource associated with a form, the vocabulary term `cov:confirmable` can be used.
+                A value of `true` indicates that a CON request SHOULD be used by the Consumer, while a value of `false` indicates that a NON request SHOULD be used instead (for potential usages of the vocabulary term see [[[#example-request-con-non]]]).
+                However, as the vocabulary term is just a hint, the consumer is not bound to this recommendation.
```

Comment:
Suggestions made against this content in `index.html` also apply to this content in `index.template.html`. But I don't understand why these files contain so much duplication.